### PR TITLE
New version: joncampbell123.DOSBox-X version dosbox-x-v0.83.18

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/29247)

### CODE REVIEW

File: manifests/j/joncampbell123/DOSBox-X/dosbox-x-v0.83.18/joncampbell123.DOSBox-X.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=QUSU.7-1-4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: joncampbell123.DOSBox-X
+PackageVersion: dosbox-x-v0.83.18
```
Comment (ItzLevvie) [inline manifests/j/joncampbell123/DOSBox-X/dosbox-x-v0.83.18/joncampbell123.DOSBox-X.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.83.18
```
Well - looks like only 1.1 can save us from upgrade loops. For now, just remove the `dosbox-x-v` bit from automation.
image
image

File: manifests/j/joncampbell123/DOSBox-X/dosbox-x-v0.83.18/joncampbell123.DOSBox-X.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=QUSU.7-1-4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: joncampbell123.DOSBox-X
+PackageVersion: dosbox-x-v0.83.18
```
Comment (ItzLevvie) [inline manifests/j/joncampbell123/DOSBox-X/dosbox-x-v0.83.18/joncampbell123.DOSBox-X.installer.yaml:5]:
```suggestion
PackageVersion: 0.83.18
```
Well - looks like only 1.1 can save us from upgrade loops. For now, just remove the `dosbox-x-v` bit from automation.
image
image

File: manifests/j/joncampbell123/DOSBox-X/dosbox-x-v0.83.18/joncampbell123.DOSBox-X.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=QUSU.7-1-4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: joncampbell123.DOSBox-X
+PackageVersion: dosbox-x-v0.83.18
```
Comment (ItzLevvie) [inline manifests/j/joncampbell123/DOSBox-X/dosbox-x-v0.83.18/joncampbell123.DOSBox-X.yaml:5]:
```suggestion
PackageVersion: 0.83.18
```
Well - looks like only 1.1 can save us from upgrade loops. For now, just remove the `dosbox-x-v` bit from automation.
image
image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by vedantmgoyal2009** [comment]
@ItzLevvie Should I add a note in README that Packages delivered through GitHub Releases are auto-updated so users don't create PRs to update them and instead find other packages that need to be updated in the repo?

**Pull request merged**